src/gpu/intel/ocl/micro_sdpa.cpp

```diff
@@ -500,6 +500,9 @@ status_t micro_sdpa_t::init(impl::engine_t *engine) {
         kernel_ctx.define_int("PREFETCH_D_MAX", nstl::min(pd()->d_max(), 64));
     }
 
+    kernel_ctx.define_int("SAFE_SOFTMAX",
```

Comment:
I used `SOFTMAX_INF_AS_ZERO` to be more specific on the algorithm name. As Mourad mentioned, who knows if more "safe" versions coming our way.